BangazonAPI/Controllers/PaymentTypeController.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System.Data;
+using System.Data.SqlClient;
+using BangazonAPI.Models;
+
+namespace BangazonAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PaymentTypeController : ControllerBase
+    {
+        private readonly IConfiguration _config;
+
+        public PaymentTypeController(IConfiguration config)
+        {
+            _config = config;
+        }
+
+        public SqlConnection Connection
+        {
+            get
+            {
+                return new SqlConnection(_config.GetConnectionString("DefaultConnection"));
+            }
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Get()
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = "SELECT PaymentType.Id, PaymentType.AcctNumber, PaymentType.Name, PaymentType.Archived, PaymentType.CustomerId AS 'Customer Id' FROM PaymentType LEFT JOIN Customer on PaymentType.CustomerId = Customer.Id";
```

Comment:
SQL statement looks great but the ticket doesn't call for customer information. Adjusting the SQL  query to only contain information for the PaymentType Query will make the call more efficient.

Reply:
I fixed this.